src/components/dataCard.tsx

```diff
@@ -0,0 +1,122 @@
+import React from "react";
+
+/*
+Usage :
+<DataCard data="YourParagraph|YourImageSrc"/>
+*/
+
+export default function DataCard({ data }: { data: string }) {
+  // Split text to 2 parts from | character
+  const [para, imgsrc] = data.split("|");
+
+  return (
+    <div
+      style={{
+        width: "1200px",
```

Comment:
Refactor these inline styles to use tailwind, or without repeating the same set of lines create the styles object out inside the component scope without repeating the same lines.

Reply:
ah shit mek fix krnn beriuna 
Fs in the chat, hdnnm gimme a sec

Reply:
Done